### ISSUE TITLE
Use git describe in About Dialog

### ISSUE THREAD

**Issue by nbsp** [tenacityteam/tenacity-legacy#460]
- [x] I have read the specified guidelines for issues

#396 switched to using the output of `git describe` in versions and various strings, but the About Dialog was left untouched. This is what it shows now:

![About Dialog](https://libera.ems.host/_matrix/media/r0/download/semisol.dev/eJCTyhvVYmlPpYyBHkGulrbL/image.png)

**Describe the solution you'd like**
We could use the variable `GIT_DESCRIBE`, defined in `/CMakeLists.txt`.

Also might be worth adding one more reference here, just under the logo:
![About Dialog main page](https://libera.ems.host/_matrix/media/r0/download/semisol.dev/aHLvNhIRRoppeGDCImOahrxd/image.png)

**Additional context**
Don't worry about git commits vs. actual versions; `git describe --abbrev=7` shows just the tag if it's on it.

**Comment by humrochagf** [comment]
Hello!

I would love to start contributing to the project so I started by doing some digging on the code and the first thing I found was how the commit Id gets displayed and it is generated by `cmake-proxies/cmake-modules/Version.cmake` that generates the `build\src\private\RevisionIndex.h` that contains the `REV_LONG` ref to the build commit. This is also used to build the link to the actual reference commit on github when you click on the commit id.

My firs question is, should we keep the commit id and add another line on top named version? And then it would be something like this:

```
Version: [the content of git describe]
Commit Id: [keep as it is]
...
```

Or Commit Id should be removed and replaced by version?
